### PR TITLE
Feat/BoardPage, List and Task components

### PR DESCRIPTION
**Pull request by allenanle**

### CODE REVIEW

File: client/src/components/BoardPage.jsx [hunk 1]
```diff
@@ -1,10 +1,64 @@
...
+  }
+
+  onCreateList() {
+    this.props.createList(this.state.listName, this.props.board_id)
+  }
+
+  render() {
+    return (
+      <div>
+        <h3>{this.props.boardname}</h3>
```
Comment (cdey) [inline client/src/components/BoardPage.jsx]: Forgot padding

File: client/src/components/Lobby.jsx [hunk 2]
```diff
@@ -29,16 +27,17 @@ export class BoardListPage extends React.Component {
...
+        <input
+          value={ this.state.boardName }
           onChange={ this.handleBoardNameChange }
         />
-        <button onClick={ this.handleSubmit } >Create Board</button>
+        <button onClick={ this.handleSubmit }>CREATE BOARD</button>
         { this.props.boardlist.map((board) => (
-          <div 
+          <div
+            key={board.boardname}
```
Comment (cdey) [inline client/src/components/Lobby.jsx]: Forgot padding

File: client/src/components/Task.jsx [hunk 3]
```diff
@@ -1,9 +1,9 @@
 import React from 'react'
 
-const Task = () => {
+export const Task = (props) => {
   return (
     <div>
-      Hello world!
+      . {props.text} – assigned to {props.assigned}
```
Comment (cdey) [inline client/src/components/Task.jsx]: Forgot padding

File: client/src/reducers/Task.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      return {
+        ...state,
+        error: action.error
+      }
+    case 'CREATE_TASK':
+      for (var i = 0; i < state.lists.length; i++) {
+        if (state.lists[i].id === action.list_id) {
+          return {
+            ...state.lists[i],
+            tasks: [...state.lists[i].tasks, {text: action.text, list_id: action.list_id}]
```
Comment (cdey) [inline client/src/reducers/Task.js]: PAD

File: client/src/reducers/List.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+      }
+    case 'CREATE_ERROR':
+      return {
+        ...state,
+        error: action.error
+      }
+    case 'CREATE_LIST':
+      return {
+        ...state,
+        lists: [...state.lists, {listname: action.listname, board_id: action.board_id, tasks: []}]
```
Comment (cdey) [inline client/src/reducers/List.js]: DING

**Review by ejk91 — APPROVED** [review]

**Comment by ejk91** [comment]
Use better naming for your git branch!
Also provide a description of your PR so it gives us context of what you did.

nice syntax

File: client/src/components/BoardPage.jsx (hunk 1 above)
Comment (ejk91) [reply client/src/components/BoardPage.jsx]: Forgot padding

File: client/src/components/Lobby.jsx (hunk 2 above)
Comment (ejk91) [reply client/src/components/Lobby.jsx]: Forgot padding

File: client/src/components/Task.jsx (hunk 3 above)
Comment (ejk91) [reply client/src/components/Task.jsx]: Forgot padding

File: client/src/reducers/Task.js (hunk 4 above)
Comment (ejk91) [reply client/src/reducers/Task.js]: PAD

File: client/src/reducers/List.js (hunk 5 above)
Comment (ejk91) [reply client/src/reducers/List.js]: DING

**Comment by cdey** [comment]
just checking...you rebased before this, right?

**Pull request merged**